.jenkinsci/bindings.groovy

```diff
@@ -108,7 +110,8 @@ def doAndroidBindings(abiVersion) {
     [currentPath, "\$PLATFORM", abiVersion, "\$BUILD_TYPE_A", sh(script: 'date "+%Y%m%d"', returnStdout: true).trim(), commit.substring(0,6)])
   sh """
     (cd /iroha; git init; git remote add origin https://github.com/hyperledger/iroha.git; \
-    git fetch --depth 1 origin develop; git checkout -t origin/develop)
+    # TODO possible race on consequent pushes to the same branch
+    git fetch --depth 1 origin ${env.GIT_LOCAL_BRANCH}; git checkout -t origin/${env.GIT_LOCAL_BRANCH})
```

Comment:
This will fail if built from PR:
![image](https://user-images.githubusercontent.com/8650477/41762181-9b0ce826-7602-11e8-8c8a-bc452a48ae49.png)
I suggest to change it to: `git fetch origin ${GIT_COMMIT}; git checkout FETCH_HEAD)`. 
I tested it and it works OK: https://jenkins.soramitsu.co.jp/job/iroha/job/iroha-hyperledger/job/fix%252Ffix-bindings-build/4/consoleFull
![image](https://user-images.githubusercontent.com/8650477/41762346-0e2a07ee-7603-11e8-9b5b-b8331877cd10.png)